### PR TITLE
Update explanation for handling magic comments

### PR DESCRIPTION
**Pull request by bgeuken**
The previous explanation was a bit wordy and thus harder to
understand, eg. if there is no shebang, it should be clear that it
doesn't have to be considered:-)

Also it sounded like 'if they' would reference to magic comments, while
it actually meant a possible shebang.

### CODE REVIEW

**Comment by mikegee** [comment]
Consider:

> , but below the shebang in executable scripts.

to avoid implying the shebang is required in all Ruby files.

**Comment by bbatsov** [comment]
Good point, @mikegee!

**Comment by pirj** [comment]
This is pretty much covered in https://github.com/rubocop-hq/ruby-style-guide#below-shebang
